tools/importer-rest-api-specs/components/terraform/testing/dependencies_template.go

```diff
@@ -158,6 +158,9 @@ resource "%[1]s_kubernetes_cluster" "test" {
     node_count = 1
     vm_size    = "Standard_DS2_v2"
   }
+  upgrade_settings {
+      max_surge = "10%%"
```

Comment:
since this is being output into another Sprintf function (in the TF generator) I suspect this might need another `%` here?

```suggestion
      max_surge = "10%%%"
```

Reply:
good point, it actually needs 2 additional %